lambda/app/src/bceid-webservice-proxy/idir.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { createIdirUser } from '../keycloak/users';
+
+export const searchIdirUsers = async (bearerToken: string, { field, search }: { field: string; search: string }) => {
+  const results = await axios
+    .get(`${process.env.BCEID_WEBSERVICE_PROXY}/idir`, {
+      params: { field, search },
+      headers: { Authorization: `Bearer ${bearerToken}` },
+    })
+    .then((res: any) => res.data);
+
+  return results;
+};
+
+export const importIdirUser = async (bearerToken: string, { guid, userId }: { guid: string; userId: string }) => {
+  const results = await axios
+    .get(`${process.env.BCEID_WEBSERVICE_PROXY}/idir`, {
+      params: { field: 'userId', search: userId },
+      headers: { Authorization: `Bearer ${bearerToken}` },
+    })
+    .then((res: any) => res.data);
+
+  if (!results || results.length === 0) return false;
+  const result = results.find((user) => user.guid === guid);
+  if (!result) return false;
+
+  console.log({
+    guid: result.guid,
+    userId: result.userId,
+    email: result.contact.email,
+    firstName: result.individualIdentity.name.firstname,
+    lastName: result.individualIdentity.name.surname,
+    displayName: result.displayName,
+  });
+
+  const aa = await Promise.all(
```

Comment:
Is it possible to get a more descriptive name for this constant? Or is this a standard name for a lambda function constant?

Reply:
right, forgot to remove the constant; I removed the variable.